cdkworkshop.com/cdkworkshop.com.ts

```diff
@@ -159,7 +133,7 @@ export class CdkWorkshop extends Stack {
                         },
                         contentSecurityPolicy: {
                             override: true,
-                            contentSecurityPolicy: `default-src 'self'; connect-src 'self' *.shortbread.aws.dev *.google-analytics.com; script-src 'self' *.google-analytics.com ${inlineScriptSha}; img-src 'self' *.shortbread.aws.dev; style-src 'unsafe-hashes' 'self' ${inlineStyleSha};`,
+                            contentSecurityPolicy: cspPolicyContent,
```

Comment:
The new policy removes `img-src` is that ok?

Reply:
Using `default-src` sets a value for any fields that don't have a value otherwise.